### ISSUE TITLE
Bulleted Lists

### ISSUE THREAD

**Issue by barleedar** [soflyy/oxygen-bugs-and-features#94]
I would like to see bullet points included without having to add them via Text editor.

**Comment by fscriven** [comment]
Would be nice too if you could e.g. take a paragraph/sentence you have created and use it as list item inside a list. Ability to hide list items vital for me.

**Comment by avocadesign** [comment]
I totally agree - it's the one thing that I think would be really jarring for clients to edit when the rest with client control turned on is quite straight forward
